assets/templates/partials/cards/census-data.tmpl

```diff
@@ -11,6 +11,13 @@
                         <a href="/search{{ .Data.GetCensusDataURLQuery }}" class="ons-list__link">{{ localise "GetCensusDataLink1" .Language 1 }}</a>
                     {{ end }} 
                 </li>
+                {{ if .Data.EnableCustomDataset }}
```

Comment:
Would it be wise to extend component tests to check if flag is on that this card is displayed?